helpers/types/common.ts

```diff
@@ -2,7 +2,9 @@ import { HardhatEthersHelpers } from '@nomiclabs/hardhat-ethers/types'
 import { ethers, providers } from 'ethers'
 import { HardhatRuntimeEnvironment } from 'hardhat/types'
 
-export type ValueOf<T> = T[keyof T]
+export type PartialRecord<K extends keyof any, T> = {
+  [P in K]?: T
+}
```

Comment:
```suggestion

```
There is absolutely no need of that. You can achieve the same thing with the native TS types:
`Partial<Record<K,V>>`